CONTRIBUTING.md

```diff
@@ -2,15 +2,216 @@ Guidelines for Contributing
 ===========================
 
 Welcome to the D community and thanks for your interest in contributing!
-If this is your first pull request, please read [Contributing to Phobos](https://wiki.dlang.org/Contributing_to_Phobos).
-Have a look at [Starting as a Contributor](http://wiki.dlang.org/Starting_as_a_Contributor#Building_D) for build instructions.
-
-More Links
-----------
-
-* Fork [on Github](https://github.com/dlang/phobos)
-* Use our [Bugzilla bug tracker](https://issues.dlang.org)
-* Follow the [D style](http://dlang.org/dstyle.html)
-* Participate in [our forum](http://forum.dlang.org/)
-* Ask questions on our `#d` IRC channel on freenode.org ([web interface](https://kiwiirc.com/client/irc.freenode.net/d))
-* Review Phobos additions in the [Review Queue](http://wiki.dlang.org/Review_Queue).
+If you need help you can ask questions on `#d` IRC channel on
+freenode.org ([web interface](https://kiwiirc.com/client/irc.freenode.net/d))
+or on [our forum](http://forum.dlang.org/).
+
+Guidelines for contributing to Phobos
+-------------------------------------
+
+The usual workflow is:
+
+1.  [Fork](https://help.github.com/articles/fork-a-repo/)
+    [Phobos](https://github.com/D-Programming-Language/phobos) on Github
+2.  Create your [own branch](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches)
+3.  [Work locally](https://wiki.dlang.org/Starting_as_a_Contributor#Building_D) on
+    your new feature or fix (see the tips below)
+4.  [Test](#tests) your improvements locally
+5.  Submit your [pull request](https://help.github.com/articles/creating-a-pull-request/) (PR)
```

Comment:
I would not extend so much on such a trivial setup. I'd just go for something along the lines of "Contributing to Phobos can be done using the [usual Github workflow](https://guides.github.com/introduction/flow/)."
The point of this guide is to explain how to contribute to this repository, not how to use Github (I know one might want to provide minimum information here, but IMO one sentence is enough).